scripts/processing_manifest.py

```diff
@@ -69,51 +77,75 @@ class Acquisition(AindModel):
 
 
 class ChannelRegex(Enum):
+    """Regular expression for the channel imaging data"""
+
     CHANNEL_NAME = r"Ex_\d{3}_Em_\d{3}"
 
 
 class AxisResolution(AindModel):
+    """Axial resolution for the channel imaging data"""
+
     axis_name: AxisName = Field(..., title="Axis name")
     unit: SizeUnit = Field(SizeUnit.UM, title="Axis physical units")
     resolution: float = Field(..., title="Axis resolution")
 
 
 class StitchingParameters(AindModel):
```

Comment:
Having classes that inherit AindModel outside of the aind-data-schema repo seems counter-intuitive. You can just inherit pydantic's BaseModel to create your own pydantic model. They also have a BaseSettings class that is a useful data structure to contain parameters.